packages/react/src/primitives/Field/FieldDescription.tsx

```diff
@@ -8,12 +9,15 @@ export const FieldDescription: React.FC<FieldDescriptionProps> = ({
   descriptiveText,
   labelHidden,
   ...rest
-}) => {
-  return !labelHidden && descriptiveText ? (
-    <Text className={ComponentClassNames.FieldDescription} {...rest}>
-      {descriptiveText}
-    </Text>
-  ) : null;
-};
+}) => (
+  <Text
+    className={classNames(ComponentClassNames.FieldDescription, {
+      'amplify-visually-hidden': labelHidden,
```

Comment:
This could be `ComponentClassNames.VisuallyHidden`

Reply:
Done